README.md

```diff
@@ -9,3 +9,481 @@ https://github.com/docker-library/mongo
 
 # Disclaimer
 This is not an official Google product.
+
+# <a name="about"></a>About
+
+This image contains an installation of MongoDB
+
+For more information, see the
+[Official Image Marketplace Page](https://console.cloud.google.com/marketplace/product/google/mongodb4).
+
+Pull command (first install [gcloud](https://cloud.google.com/sdk/downloads)):
```

Comment:
I'd split this section, here and in other READMEs:
```
    ### Prerequisites

    Configure [gcloud](https://cloud.google.com/sdk/gcloud/) as a Docker credential helper:

    ```shell
    gcloud auth configure-docker
    ```

    ### Pull command

    ```shell
    docker -- pull marketplace.gcr.io/google/mongodb4
    ```
```